criu/cr-restore.c

```diff
@@ -3633,7 +3633,13 @@ static int sigreturn_restore(pid_t pid, struct task_restore_args *task_args, uns
 	 * it gets unmapped at the very end of __export_restore_task
 	 */
 
-	task_args->proc_fd = dup(get_service_fd(PROC_FD_OFF));
+	i = get_service_fd(PROC_FD_OFF);
```

Comment:
pls don't use random variables here. This doesn't look right. It is better to wrap this in a function.